### PR TITLE
easyloggingpp: add options

### PR DESCRIPTION
**Pull request by geirhei**
Specify library name and version: easyloggingpp/9.97.0

Exposes more configuration options to conan.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/README.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/easyloggingpp/all/conanfile.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import os
-from conans import ConanFile, CMake, tools
+from conan.tools.cmake import CMake
```
[the remark this answers is not in the document]
Comment (geirhei) [reply recipes/easyloggingpp/all/conanfile.py]: Thanks

**Review by uilianries — APPROVED** [review]
LGTM

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
you should really contribute this CMake build script to upstream.

https://github.com/amrayn/easyloggingpp/blob/8489989bb26c6371df103f6cbced3fbee1bc3c2f/CMakeLists.txt#L61 does not even support these options and they are compiled in the .cc 🤷 

Like this is required because of how CCI is shipping the lib but it does not match upstream which is really confusing

**Comment by geirhei** [comment]
> you should really contribute this CMake build script to upstream.
> 
> https://github.com/amrayn/easyloggingpp/blob/8489989bb26c6371df103f6cbced3fbee1bc3c2f/CMakeLists.txt#L61 does not even support these options and they are compiled in the .cc shrug
> 
> Like this is required because of how CCI is shipping the lib but it does not match upstream which is really confusing

Made as issue for that upstream: https://github.com/amrayn/easyloggingpp/issues/826

Unfortunately, the project does not seem to be maintained anymore.

**Pull request merged**